src/js/components/FormField/FormField.js

```diff
@@ -502,7 +509,20 @@ const FormField = forwardRef(
         }}
         onBlur={(event) => {
           setFocus(false);
-          if (contextOnBlur) contextOnBlur(event);
+
+          // if input has a drop and focus is within drop
+          // prevent onBlur validation from running until
+          // focus is no longer within the drop or input
+          const dropPortalId = withinDropPortal(event.relatedTarget);
+          if (
+            (dropPortalId === undefined ||
+              portalContext.indexOf(parseInt(dropPortalId, 10)) !== -1) &&
```

Comment:
Can we add a small comment speaking to what portalContext is doing here?

Reply:
Yes, some code moved around after I incorporated Eric's feedback so I ended up adding a comment in the utils/dom.js file